app/controllers/support_interface/provider_users_controller.rb

```diff
@@ -10,8 +10,9 @@ def new
 
     def create
       @form = ProviderUserForm.new(provider_user_params)
+      service = InviteProviderUser.new(provider_user_form: @form)
```

Comment:
Hmm `ProviderUserForm#save` is also used for the edit/update cycle. I think I prefer using `InviteProviderUser` service directly for new/create. Besides, I see the service as more general than the form (it will soon send emails, too). If you feel calling the service from the form#save is better, we can do that but will need to add logic to `#save` to determine if the service should be called.

Reply:
Ok, we've got a way forward after chatting IRL. The controller will ask the form to build a ProviderUser model, which will then be passed to the service. The service will attempt to save this model, if it cannot the errors will be fed back to the user through the form. We are in the process of merging https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1145 in first, so `first_name` and `last_name` will be available on the model.

In the presence of API-related errors, we'll stop handling the Exception within the service and let the controller handle it, in order to feed back these errors to the UI.